invokeai/frontend/web/src/features/system/components/AboutModal/AboutModal.tsx

```diff
@@ -0,0 +1,140 @@
+import { ExternalLinkIcon } from '@chakra-ui/icons';
+import {
+  Flex,
+  Grid,
+  GridItem,
+  Image,
+  Link,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+  useDisclosure,
+} from '@chakra-ui/react';
+import { InvHeading } from 'common/components/InvHeading/wrapper';
+import {
+  InvModal,
+  InvModalBody,
+  InvModalCloseButton,
+  InvModalContent,
+  InvModalFooter,
+  InvModalHeader,
+  InvModalOverlay,
+} from 'common/components/InvModal/wrapper';
+import { InvText } from 'common/components/InvText/wrapper';
+import ScrollableContent from 'common/components/OverlayScrollbars/ScrollableContent';
+import { fromPairs, map, toPairs } from 'lodash-es';
+import InvokeLogoYellow from 'public/assets/images/invoke-tag-lrg.svg';
+import type { ReactElement } from 'react';
+import { cloneElement, memo, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  useGetAppDepsQuery,
+  useGetAppVersionQuery,
+} from 'services/api/endpoints/appInfo';
+
+type AboutModalProps = {
+  /* The button to open the Settings Modal */
+  children: ReactElement;
+};
+
+const AboutModal = ({ children }: AboutModalProps) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { t } = useTranslation();
+  const { data: appDeps } = useGetAppDepsQuery();
+  const { data: appVersion } = useGetAppVersionQuery();
+  const appDeps_arr = map(toPairs(appDeps), (d) => fromPairs([d]));
```

Comment:
This is a bit convoluted. Let's leverage RTK Query to make this simpler:

```ts
  const { deps } = useGetAppDepsQuery(undefined, {
    selectFromResult: ({ data }) => ({
      deps: data ? map(data, (version, name) => ({ name, version })) : [],
    }),
  });
```

Then you can map over `deps` easily:
```ts
deps.map(({ name, version }, i) => (
  // ... do something with name and version
))
```